FlightStreamDeck.Logics/ImageLogic.cs

```diff
@@ -15,7 +16,12 @@ public interface IImageLogic
         string GetNumberImage(int number);
         string GetNavComImage(string type, bool dependant, string value1 = null, string value2 = null, bool showMainOnly = false);
         public string GetHorizonImage(float pitchInDegrees, float rollInDegrees, float headingInDegrees);
-        public string GetGaugeImage(string text, float value, float min, float max);
+        public string GetGenericGaugeImage(string text, float value, float min, float max);
+        public string GetCustomGaugeImage(string textTop, string textBottom,
```

Comment:
Maybe either 1 variable for each line, or all variables on 1 line, but grouped by two I haven't seen before :)

Reply:
I usually put one line per group of parameters also :P.